engine-cockpit/webContent/resources/composite/DynamicConfig.xhtml

```diff
@@ -227,12 +227,15 @@
       </ui:repeat>
     </h:form>
 
-  <p:dialog header="#{ivy.cm.co('/dynamicConfig/ConnectionTestUrlDialogHeader')} #{property.value}"
+  <p:dialog header="#{ivy.cm.co('/dynamicConfig/ConnectionTestUrlDialogHeader')} #{tlsTesterBean.uri}"
     id="connectionTestModel" width="70%" widgetVar="connectionTestModel"
     modal="true" responsive="false" closeOnEscape="true">
+    <h:panelGroup style="height:150px" id="missingCertView" rendered="#{tlsTesterBean.hasMissingCerts()}">
+      <jc:TlsTesterMissingCertView id="missing" bean="#{tlsTesterBean}" cert="#{tlsTesterBean.missingCert}" />
+    </h:panelGroup>
```

Comment:
Second: the missing cert view was not part of the ldap SSL tester.
Is this by intention or was this part just forgotten?